content/en/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -889,6 +893,322 @@ app:
+
+```yaml
+otel:
+  enabled: true
+  daemonset:
+    config:
+      exporters:
+        otlphttp:
+          endpoint: "https://example.com:4318"  # Required - your OTEL collector endpoint
+          headers:
+            api-key: "<API-KEY>"  # Add any required authentication headers
+      service:
+        pipelines:
+          traces:
+            receivers: [otlp]
+            processors: [memory_limiter, batch]
+            exporters: [otlphttp]
+          metrics:
+            receivers: [otlp]
+            processors: [memory_limiter, batch]
+            exporters: [otlphttp]
+          logs:
+            receivers: [otlp]
+            processors: [memory_limiter, batch]
+            exporters: [otlphttp]
+```
+
+#### Use Kubernetes secrets for sensitive data
+
+For production deployments, use Kubernetes secrets to store sensitive information like API keys:
+
+```yaml
+otel:
+  enabled: true
+  daemonset:
+    config:
+      exporters:
+        otlphttp:
+          endpoint: "https://example.com:4318"
+          headers:
+            api-key: "${OTEL_API_KEY}"  # Reference environment variable
+    extraEnvs:
+      - name: OTEL_API_KEY
+        valueFrom:
+          secretKeyRef:
+            name: otel-secrets
+            key: api-key
+```
+
+Create the secret before deploying:
+```bash
+kubectl create secret generic otel-secrets --from-literal=api-key=<YOUR-API-KEY>
+```
+
+#### Configure queue and retry behavior
+
+Configure queue and retry behavior for reliable data delivery:
+
+```yaml
+otel:
+  enabled: true
+  daemonset:
+    config:
+      exporters:
+        otlphttp:
+          endpoint: "https://example.com:4318"
+          headers:
+            api-key: "${OTEL_API_KEY}"
+          sending_queue:
+            enabled: true
+            num_consumers: 10      # Number of parallel workers sending data
+            queue_size: 1000       # Maximum number of batches in the queue
+          retry_on_failure:
+            enabled: true
+            initial_interval: 5s   # Time to wait before first retry
+            max_interval: 30s      # Maximum time between retries
+            max_elapsed_time: 300s # Maximum total time spent retrying
+```
+
+#### Configure processors
+
+Processors modify telemetry data between collection and export. Common processors include batching for efficiency and memory limiting for resource management:
+
+```yaml
+otel:
+  enabled: true
+  daemonset:
+    config:
+      processors:
+        batch:
+          timeout: 5s                # Time to wait before sending a batch
+          send_batch_size: 10000     # Number of spans/metrics/logs per batch
+          send_batch_max_size: 20000 # Maximum batch size
+        memory_limiter:
+          check_interval: 5s         # How often to check memory usage
+          limit_percentage: 80       # Maximum memory usage as percentage of total
+          spike_limit_percentage: 25 # Tolerated spike in memory usage
+        k8sattributes:
```

Comment:
```suggestion
        k8sattributes:
```
Add a comment (please keep the comment alignment, it helps with readability) that this is a processor for enriching data before export (if that is accurate). Context: https://opentelemetry.io/docs/platforms/kubernetes/collector/components/#kubernetes-attributes-processor